packages/i18n/locales/en/translation.json

```diff
@@ -224,20 +229,22 @@
     "invalidExecuteMessage": "Invalid execute message",
     "invalidMessage": "Invalid message",
     "loadingData": "Failed to load data.",
+    "meTransactionInvalid": "Correct the errors above before previewing or executing.",
     "missingGovernanceTokenAddress": "Missing governance token address.",
     "mustBeMemberToCreateCompensationCycle": "You must be a member of the DAO to create a compensation cycle.",
     "mustBeMemberToCreateProposal": "You must be a member of the DAO to create a proposal.",
     "mustBeMemberToCreateVestingPayment": "You must be a member of the DAO to create a vesting payment.",
     "nameIncorrectLength": "A DAO's name must be between {{min}} and {{max}} characters.",
     "needsVote": "You must select a vote.",
     "nftCollectionNotChosen": "You must create an NFT collection or enter the address of an existing collection before using this action.",
+    "nftLoadFailedReopenModal": "Failed to load NFTs. Reopen the modal to try again.",
```

Comment:
When would a user see this error?

Reply:
This used to mean you rejected Stargaze's wallet enable request, but I made it convert from Juno's address, so it no longer asks that. This will now show when a query fails to the chain/indexer/Stargaze API, so it should be more like the other 'network issue try refreshing' errors instead. Reopening the modal won't do anything anymore.

Reply:
Changed to `Check your internet connection, refresh the page, or try again later.`

Is that good?

Reply:
Yes!